python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1439,6 +1439,444 @@ keys:
                   description: |
                     Routes learned by the OSPF protocol.
                     The `ospf_route_type` is valid for source_protocols 'ospf' and 'ospfv3'.
+      address_family_ipv4_labeled_unicast:
+        type: dict
+        keys:
+          aigp_session:
+            type: dict
+            keys:
+              confederation:
+                type: bool
+              ebgp:
+                type: bool
+              ibgp:
+                type: bool
+          bgp:
+            type: dict
+            keys:
+              additional_paths:
+                type: dict
+                description: BGP additional-paths commands.
+                keys:
+                  receive:
+                    type: bool
+                    description: Receive multiple paths.
+                  send:
+                    type: dict
+                    description: Send multiple paths.
+                    keys:
+                      any:
+                        type: bool
+                        description: Any eligible path.
+                      backup:
+                        type: bool
+                        description: Best path and installed backup path.
+                      ecmp:
+                        type: bool
+                        description: All paths in best path ECMP group.
+                      ecmp_limit:
+                        type: int
+                        description: Amount of ECMP paths to send.
+                        convert_types:
+                          - str
+                        min: 2
+                        max: 64
+                      limit:
+                        type: int
+                        description: Amount of paths to send.
+                        convert_types:
+                          - str
+                        min: 2
+                        max: 64
+              missing_policy:
```

Comment:
The same schema of missing policy defined under peer group can be used here also as the include option is missing here

Reply:
I'm not sure I follow - the `include` options are handled by the boolean flags, but still require a direction either way, so fall within the direction in/out scope e.g. `neighbor 1.2.3.4 missing-policy include community-list prefix-list sub-route-map direction out action deny`.

Reply:
But from the below schema and j2 template we are not able to configure the `include` for router_bgp. address_family_ipv4_labeled_unicast .bgp.

```
router bgp 65102
   address-family ipv4 labeled-unicast
      bgp missing-policy include community-list prefix-list sub-route-map direction out action deny
leaf1-dc1(config-router-bgp-af-label)#
```

I was trying to say that we can use the same schema of missing_policy here in `router_bgp. address_family_ipv4_labeled_unicast.bgp` which you used in `router_bgp. address_family_ipv4_labeled_unicast.peer_groups`

Reply:
Ah, gotcha